src/test/java/edu/hm/hafner/analysis/parser/ParserTester.java

```diff
@@ -10,41 +10,77 @@
 import edu.hm.hafner.analysis.Issue;
 import edu.hm.hafner.analysis.IssueBuilder;
 import edu.hm.hafner.analysis.Priority;
-import edu.hm.hafner.util.Ensure;
-import static org.junit.jupiter.api.Assertions.*;
+import static edu.hm.hafner.analysis.assertj.Assertions.*;
```

Comment:
Die Klasse bitte nicht im PR ändern, sonst gibt es merge Conflict.

Reply:
Ok, ich war mir nicht sicher ob diese Klasse bearbeitet werden soll. Sie taucht zumindest in der google-docs Liste auf. Hab daher noch für "Pep8ParserTest" committet um auf insgesamt 4 Parser-Tests zu kommen.

Muss ich die Änderungen an ParserTest im PR rückgängig machen oder müsste ich dazu einen neuen PR erstellen?

Reply:
Am besten die Datei wieder so herstellen wie vorher und ein neues Commit+Push, dann sollte die raus sein...

Reply:
Hab's reverted.